vision/cloud-client/detect/beta_snippets.py

```diff
@@ -0,0 +1,350 @@
+# limitations under the License.
+
+"""Demonstrates beta features using the Google Cloud Vision API.
+
+Example usage:
+  python beta_snippets.py web-entities resources/city.jpg
+  python beta_snippets.py detect-document resources/text.jpg
+  python beta_snippets.py safe-search resources/wakeupcat.jpg
+  python beta_snippets.py web-detect resources/landmark.jpg
+"""
+
+# [START imports]
+import argparse
+import io
+
+from google.cloud import vision_v1p1beta1 as vision
+# [END imports]
+
+
+# [START vision_web_entities_uri]
+def web_entities_uri(uri):
+    client = vision.ImageAnnotatorClient()
+
+    image = vision.types.Image()
+    image.source.image_uri = uri
+
+    response = client.web_detection(image=image)
+
+    for entity in response.web_detection.web_entities:
+        print(u'\nDescription: {}'.format(entity.description))
+        print('Score: {}'.format(entity.score))
+# [END vision_web_entities_uri]
+
+
+# [START vision_web_entities_file]
+def web_entities_file(path):
+    client = vision.ImageAnnotatorClient()
+
+    with io.open(path, 'rb') as image_file:
+        content = image_file.read()
+
+    image = vision.types.Image(content=content)
+
+    response = client.web_detection(image=image)
+
+    for entity in response.web_detection.web_entities:
+        print(u'\nDescription: {}'.format(entity.description))
+        print('Score: {}'.format(entity.score))
+# [END vision_web_entities_file]
+
+
+# [START vision_web_entities_include_geo_results_uri]
+def web_entities_include_geo_results_uri(uri):
+    client = vision.ImageAnnotatorClient()
+
+    image = vision.types.Image()
+    image.source.image_uri = uri
+
+    web_detection_params = vision.types.WebDetectionParams(
+        include_geo_results=True)
+    image_context = vision.types.ImageContext(
+        web_detection_params=web_detection_params)
+
+    response = client.web_detection(image=image, image_context=image_context)
+
+    for entity in response.web_detection.web_entities:
+        print(u'\nDescription: {}'.format(entity.description))
+        print('Score: {}'.format(entity.score))
+# [END vision_web_entities_include_geo_results_uri]
+
+
+# [START vision_web_entities_include_geo_results_file]
+def web_entities_include_geo_results_file(path):
+    client = vision.ImageAnnotatorClient()
+
+    with io.open(path, 'rb') as image_file:
+        content = image_file.read()
+
+    image = vision.types.Image(content=content)
+
+    web_detection_params = vision.types.WebDetectionParams(
+        include_geo_results=True)
+    image_context = vision.types.ImageContext(
+        web_detection_params=web_detection_params)
+
+    response = client.web_detection(image=image, image_context=image_context)
+
+    for entity in response.web_detection.web_entities:
+        print(u'\nDescription: {}'.format(entity.description))
+        print('Score: {}'.format(entity.score))
+# [END vision_web_entities_include_geo_results_file]
+
+
+# [START def_detect_document]
```

Comment:
Can this region tag be renamed to explicitly call out what the snippet demonstrates? Something like `confidence_score_on_text_detection`?